src/publisher/ajson_ingestor.py

```diff
@@ -78,7 +78,7 @@ def _ingest(data, force=False):
         # only update the fragment if this article version has *not* been published *or* if force=True
         update_fragment = not av.published() or force
         fragments.add(av, XML2JSON, data['article'], pos=0, update=update_fragment)
-        fragments.set_article_json(av, quiet=True)
+        fragments.set_article_json(av, quiet=force) # article-json validation occurs here
```

Comment:
what's the end result there? A silent correction where the new JSON is ignored or does it actually break the article making it invalid? Can we do better than both cases and leave the old version in place returning an error?
Or is it because `force` should save the result no matter what? In that case silent corrections should use a different parameter as their intent is to update the article in safety, not no matter what's the result.